src/main/java/Player.java

```diff
@@ -0,0 +1,15 @@
+public class Player {
+    private boolean turnPlayed = false;
+
+    public boolean TurnPlayed(){return turnPlayed;}
+    public void playTurn()
+    {
+        turnPlayed = true;
+    }
+
+    public void endTurn()
+    {
+        turnPlayed = false;
+    }
+
+}
```

Comment:
Try and keep 1 method "takeTurn" as shown in the PDF. Also missing attribute name